pkg/utils/serviceaccount.go

```diff
@@ -91,15 +92,17 @@ var (
 	}
 )
 
-func CreateMissingPermissionForSA(ctx context.Context, kubernetesClient kubernetes.Interface, logger *zap.Logger) {
+func CreateMissingPermissionForSA(ctx context.Context, kubernetesClient kubernetes.Interface, logger *zap.Logger, mgr manager.Interface) {
```

Comment:
Please dont pass mgr to this function. We will need to see appropriate logic and then fix

Reply:
done